#! /bin/sh /usr/share/dpatch/dpatch-run
## 10_ps.1.dpatch by  <csmall@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: ps.1 normal indent widths

@DPATCH@
diff -urNad procps-3.2.7~/ps/ps.1 procps-3.2.7/ps/ps.1
--- procps-3.2.7~/ps/ps.1	2006-06-26 09:45:43.000000000 +1000
+++ procps-3.2.7/ps/ps.1	2006-06-26 12:11:58.000000000 +1000
@@ -18,12 +18,6 @@
 .nh
 .if n .ss 12 0
 .\"
-.\" See /usr/share/groff/current/tmac/an-old.tmac for what these do.
-.\" Setting them to zero provides extra space, but only do that for
-.\" plain text output. PostScript and such will remain indented.
-.if n .nr IN 0n
-.if n .nr an-prevailing-indent 0n
-.\"
 .\"
 .\" ColSize is used for the format spec table.
 .\" It's the left margin, minus the right, minus
